ci-operator/config/openshift/csi-driver-nfs/openshift-csi-driver-nfs-master.yaml

```diff
@@ -1,17 +1,26 @@
 base_images:
-  os:
+  base:
     cluster: https://api.ci.openshift.org
-    name: centos
-    namespace: openshift
-    tag: "7"
+    name: "4.5"
+    namespace: ocp
+    tag: base
 binary_build_commands: go build -o ./bin/nfsplugin ./cmd/nfsplugin
 build_root:
   image_stream_tag:
     cluster: https://api.ci.openshift.org
     name: release
     namespace: openshift
-    tag: golang-1.11
+    tag: golang-1.13
 canonical_go_repository: github.com/kubernetes-csi/csi-driver-nfs
+images:
+- dockerfile_path: images/Dockerfile.openshift.ci
+  from: base
+  inputs:
+    root:
+      as:
+      - "0"
```

Comment:
This is generally incorrect, why are you doing this?  Note that your Dockerfile that you build here should be your official one (the one used in OSBS), not a CI only image.

If the other PR had this issue, we need to fix that too.

Reply:
It comes from here: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-release-4.1.yaml#L14-L22
I just copied the image definition. Should I remove the "inputs"?

Reply:
Done :)